luda/editor/client/src/type.ts

```diff
@@ -23,3 +23,40 @@ export type CameraAngle = {
   source01Rect: XywhRect;
   dest01Rect: XywhRect;
 };
+
+export type SubtitleClip = BaseClip & {
+  type: "subtitle";
+  subtitle: Subtitle;
+};
+
+export type Subtitle = {
+  text: string;
+  fontType: SubtitleFontType;
+  style: SubtitleStyle;
+};
+export type SubtitleFontType = {
+  serif: boolean;
+  size: SubtitleFontSize;
+  language: Language;
+  fontWeight: FontWeight;
+};
+export type SubtitleStyle = {
+  color: Float32Array;
```

Comment:
Let's export `Color` from `namui/src/index.ts` like Paint and use it.

Reply:
Not CanvasKit.Color?

Reply:
Check `namui/src/index.ts` 's Paint type, how it export Paint of canvaskit.

```
export { Paint } from "canvaskit-wasm";

Reply:
Ah. Got it.
Reexported it from engin's index.ts
with https://github.com/NamseEnt/namseent/pull/18/commits/cbc870c876c447dfb4c4eee75151e5fcad4d88c3